core/locale/application/date_time_service_test.go

```diff
@@ -0,0 +1,133 @@
+package application
+
+import (
+	"flamingo.me/flamingo/v3/core/locale/domain"
+	"testing"
+	"time"
+
+	"flamingo.me/flamingo/v3/framework/flamingo"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDateTimeService_GetDateTimeFormatterFromIsoString(t *testing.T) {
+	start := time.Now()
+	now, err := time.Parse(time.RFC3339, start.Format(time.RFC3339))
+	assert.NoError(t, err, "no error received")
+
+	// error for invalid time
+	formatter, err := new(DateTimeService).GetDateTimeFormatterFromIsoString("invalid time")
+	assert.Nil(t, formatter, "got nil for formatter")
```

Comment:
The third parameter is the error message. So better "expected nil for invalid time" is a better mesage.
Same is with every message in the assert.* usage